ddtrace/profiling/exporter/pprof.pyx

```diff
@@ -437,9 +437,9 @@ class PprofExporter(exporter.Exporter):
                     convert_fn(
                         lock_name,
                         thread_id,
+                        thread_name,
```

Comment:
is it maybe worth doing something like 

```
                for event, l_events in self._group_lock_events(lock_events):
                    convert_fn(*event, list(l_events), sampling_ratio_avg)
```

or maybe named positional args?

Reply:
Yeah I think I'll do that next time since it's getting more and more complicated.